---
title: Michael Jahn, PhD
layout: page
feature_image: "/png/header/header_treemap_bw_blur.png"
feature_text: HOME
---

<img src="/png/profile_2020_circle.png" width="25%" align="right"/>

I'm a postdoctoral researcher in the group of Prof. [Paul Hudson](https://www.kth.se/profile/huds/page/2-group-members) at **KTH**, the **Royal Institute of Technology** in Stockholm, Sweden. Our group works at the **Science for Life Laboratory**, a joint institute of four different universities of the Stockholm area. I work on different projects evolving around **microbial systems biology**, mainly with photo- and lithoautotrophic model organisms, such as cyanobacteria or the 'Knallgas bacterium' *Cupriavidus necator*. 

My primary field of interest is microbial biotechnology, in the widest sense. I like to work at the interphase of experimental and computational biology. I am an enthusiastic user of R and python for statistics, metabolic modeling, and data visualization. I'm also advocating for openness and transparency in the life science. This website bundles information and resources of my scientific projects.

### Quick links

- [Scientific projects](projects)
- [Publications](publications)
- [Biography](biography)
- [Blog](blog)

### Social media + platforms

- [Google Scholar](https://scholar.google.co.uk/citations?user=jApo8PEAAAAJ)
- [Orcid](https://orcid.org/0000-0002-3913-153X)
- [github](https://github.com/m-jahn/)
- [Twitter](https://twitter.com/mich_jahn)
- [KTH home page](https://www.kth.se/profile/mjahn)
- [Research gate](https://www.researchgate.net/profile/Michael_Jahn)


### News

#### 2021-11-11 | Finally published in eLife

Our new paper, *"Protein allocation and utilization in the versatile chemolithoautotroph Cupriavidus necator"*, [was finally published in eLife!](https://elifesciences.org/articles/69019). It was a long journey, taking almost two and a half years to submission and another six months of review and improvements to finally publish it. We combined several layers of genome-wide screens: MS proteomics, gene essentiality from a transposon mutant library, and [resource balance analysis](https://sysbioinra.github.io/RBApy/) modeling. Combining these layers allowed us to show that *C. necator* is a bacterium in the making. It has several conflicting layers of regulation to tune gene expression of its highly versatile but complicated carbon and energy metabolism. Our results show for example that cells "waste" protein resources for autotrophic lifestyle even when no hydrogen is available.

#### 2021-03-24 | New preprint for *Cupriavidus* project

Our new paper about **protein utilization in the versatile lithoautotroph *Cupriavidus necator*** (aka *Ralstonia eutropha*) was [just deposited on BioRxiv.org](https://www.biorxiv.org/content/10.1101/2021.03.21.436304v1). Integrating data from proteomics, resource allocation modeling, and a transposon mutant library, we found some really interesting things. For example, of the three chromosomes that *C. necator* owns, one encodes 80% of the protein mass (the housekeeping chromosome). The other chromosomes encode specialized functions like CO2 fixation genes and were probably acquired later in evolution. The expression of these genes conflicts with overlapping functions of the ancestral chromosome.

#### 2020-09-04 | Talk at cyano conference, Porto

In a few days starts the **11th European workshop on the biology of cyanobacteria**. It was planned to take place in beautiful Porto, Portugal. I was very much looking forward to this as I haven't been on a conference for a whole year. But unfortunately COVID-19 forced the conference organizers to turn it into a digital meeting. This is not as exciting as visiting a real conference (and a new city!) and meeting people in person, but I'm nevertheless looking forward to it. I will present my work on the [cell economy of cyanobacteria](projects). I recorded my talk (15 min) and uploaded it so you can [watch it here](blog).

#### 2020-04-03 | CRISPRi library paper published     

Finally, after 4 years of work with dozens of cultivations and millions of sequenced reads, our paper about a **CRISPRi library in the model cyanobacterium *Synechocystis* was [published in Nature Communications](https://www.nature.com/articles/s41467-020-15491-7)**. What is this story about? Cyanobacteria are promising hosts for biotech production of (fine) chemicals. They are particularly interesting because they can grow solely on CO2 and sunlight. However, the productivities are low, the implemented pathways often instable, and the adaptation to varying amounts of light or CO2 not fully understood. Our group created a 'library' (a mixed pool) of thousands of mutants, each one equipped qith a CRISPR based gene repression system repressing one gene at a time. By cultivating this pool of mutants under low light, high light, and  a lactate production condition, better adapted mutants were enriched over time, and worse adapted mutants depleted. By these means we were able to find a handful of genes improving growth, acid tolerance, or lactate productivity.


#### 2019-11-08 | Awarded FORMAS grant

The Swedish funding agency FORMAS has awarded me with a grant to continue ongoing work with the fascinating litho-autotrophic bacterium *Ralstonia eutropha* a.k.a. *Cupriavidus necator*. As usual in our lab, we will study this bug and its metabolic capabilities from the systems biology perspective. Many exciting things are planned but the **focus of the project is on resource allocation**: What does a versatile soil bacterium spends its carbon and energy for? Which routes will different substrates take through the cellular metabolism? Under which conditions are enzymes saturated (under-abundance), or are not used at all (over-abundance)? We will try to answer these questions using highly controlled **steady state cultivation**, **proteomics**, **metabolic modeling**, and not yet determined **genetic engineering**. The project will officially start May 2020 and run for two years.


Last updated: 2022-01-18

-----

{% include nav-share.html %}